src/views/OrderDetail.vue

```diff
@@ -187,37 +248,59 @@ export default defineComponent({
         console.error(error)
       }
     }
-   },
+  },
   async mounted() {
     await this.getOrderDetail(this.$route.params.orderId, this.$route.params.orderPartSeqId);
     await this.getCustomerContactDetails()
+    await this.fetchRejectReasons();
   },
-  setup () {
+  setup() {
     const store = useStore();
     const router = useRouter();
 
     return {
       Actions,
-      callOutline,
+      bagHandleOutline,
+      bagRemoveOutline,
+      copyOutline,
       copyToClipboard,
+      closeCircleOutline,
+      checkmarkCircleOutline,
       hasPermission,
-      mailOutline,
       router,
       store,
-      swapVerticalOutline,
+      timeOutline,
+      mailOutline,
       sendOutline
     };
   }
 });
 </script>
 
 <style scoped>
-main {
-  max-width: 445px;
-  margin: var(--spacer-base) auto 0; 
-}
-
 .border-top {
   border-top: 1px solid #ccc;
 }
+
+@media (min-width: 768px) {
+  main {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-areas: 'left right';
+    gap: var(--spacer-base);
+    margin: var(--spacer-lg) var(--spacer-lg) 0;
+  }
+
+  aside {
+    grid-area: right;
+    /* Order Info section to stay fixed and not scrollable as there can be multiple order items which will increase the grid container size */
+    height: fit-content; 
+    position: sticky;
```

Comment:
Doesn't need to be sticky. That will fix your height problem too. Try that to simplify the code

Reply:
Yes I understand that but it is better for user to scroll the entire page here instead of both elements having different behaviors.

Reply:
Okay sir.